### PR TITLE
Changes for Appendix and Get Started

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/appendix/apis/fleet.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ tags:
   ]
 aliases:
   - /program/apis/fleet/
+  - /appendix/apis/fleet/
```
Comment (npentrel) [inline docs/appendix/apis/fleet.md:20]:
```suggestion
  - /build/program/apis/fleet/
```

File: docs/appendix/apis/fleet.md [hunk 2]
```diff
@@ -1320,9 +1321,9 @@ List the {{< glossary_tooltip term_id="module" text="modules" >}} under the orga
 {{% tab name="Python" %}}
 
 **Parameters:**
+
 - `org_id` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Organization ID for which to retrieve modules.
-- 
-**Raises:**
+- **Raises:**
```
Comment (npentrel) [inline docs/appendix/apis/fleet.md:1326]:
```suggestion

**Raises:**
```

File: docs/appendix/apis/fleet.md [hunk 3]
```diff
@@ -18,6 +18,7 @@ tags:
 aliases:
   - /program/apis/fleet/
   - /build/program/apis/fleet/
+  - /build/program/apis/fleet/
```
Comment (npentrel) [inline docs/appendix/apis/fleet.md:21]:
```suggestion
```

**Review by andf-viam — APPROVED** [review]
This LGTM! Explored the new left nav a bunch in staging and everything looks good!

Some notes, all for later consideration:
- Would be great to have a landing page for `Prepare`. Super simple, like the [Get Started landing page itself](https://docs.viam.com/get-started/), but maybe with the board carousel. I've created https://viam.atlassian.net/browse/DOCS-2464 to consider.
- Supernit: With inclusion of APIs card to Appendix, the Appendix landing page now looks a little strange with one card including a preview image, and the rest not. At some point, it'd be great to expand preview images (i.e. from front matter) to the rest of the cards, or even remove from APIs. Very minor, created https://viam.atlassian.net/browse/DOCS-2465 to consider.

**Comment by npentrel** [comment]
Thanks for creating both tickets Andrew!

**Pull request merged**